qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/IndexingIT.java

```diff
@@ -129,9 +135,9 @@ public void testNullParametersOnUpgrade() throws Exception {
     // test empty parameters
     public void testEmptyParametersOnUpgrade() throws Exception {
 
-        // Skip test if version is 1.2 or 1.3
+        // Skip test if version is 1.x.y
```

Comment:
We actually do want to test 1.0 and 1.1

Reply:
This test is not compatible with 1.0, 1.1, 1.2 or 1.3. So, I'm skipping this test if the version starts with "1."